spring-integration-core/src/main/java/org/springframework/integration/graph/IntegrationGraphServer.java

```diff
@@ -409,8 +416,7 @@ MessageGatewayNode gatewayNode(String name, MessagingGatewaySupport gateway) {
 			return new MessageGatewayNode(this.nodeId.incrementAndGet(), name, gateway, requestChannel, errorChannel);
 		}
 
-		@Nullable
-		private String channelToBeanName(MessageChannel messageChannel) {
+		private @Nullable String channelToBeanName(@Nullable MessageChannel messageChannel) {
```

Comment:
This one need to be marked with a contract:
```
@Contract("null -> null; !null -> !null")
```

See, for example, `StringUtils.quote()`.